Marlin/Configuration.h

```diff
@@ -1117,6 +1117,15 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //
 //#define SAV_3DLCD
 
+// CUNTINUE_BUTTON defines a button for user interaction in case you have no display or panel.
```

Comment:
Maybe it is better.   After all, that is the functionality being created.   But I typed that by mistake because in my head that is what I read 'CONTINUE_PIN' to mean.   We need to be consistent on the names.